confgenerator/confgenerator.go

```diff
@@ -64,7 +64,9 @@ func (uc *UnifiedConfig) GenerateOtelConfig(hostInfo *host.InfoStat) (string, er
 			Config: map[string]interface{}{
 				"user_agent": userAgent,
 				"metric": map[string]interface{}{
-					"prefix": "agent.googleapis.com/",
+					// Receivers are responsible for sending fully-qualified metric names.
+					// NB: If a receiver fails to send a full URL, OT will add the prefix `custom.googleapis.com/opencensus/`."
+					"prefix": "",
```

Comment:
Does it make sense to default to `agent.googleapis.com/` if not set? That way it errors out and developer who does not intend to send metrics to `agent.googleapis.com/` could easily tell something is not set up correctly.

Reply:
Sync'd offline. OT does not give a way to customize the default. If we override the default then we lose the behavior of only adding the prefix when the metric name isn't a URL. So NVM

Reply:
Yeah, we don't have that choice right now. Sadly.

Reply:
We are doing a rewrite of this, so we should capture this use case down in a bug/FR